.gitignore

```diff
@@ -1,2 +1,3 @@
 # Visual Studio Code
-.vscode
\ No newline at end of file
+.vscode
+captive-core*/
```

Comment:
Is this name correct?

Reply:
Yes, whenever you run captive core it creates a folder named like `captive-core-9f8g7a9f` to store the ledger files it downloads in the application's root folder. The "*" is to capture the random name given to the folder.